ginlong_solis_api_connector.py

```diff
@@ -145,17 +147,7 @@ def get_solis_cloud_data(url_part, data) -> str:
     def get_inverter_ids():
         body = '{"userid":"' + api_key_id + '"}'
         data_content = get_solis_cloud_data(endpoint_station_list, body)
-        data_json = json.loads(data_content)["data"]["page"]["records"]
-        entries = len(data_json)
```

Comment:
I think you need to merge the current changes for handling non standard device ids into your pull request.

The rest looks fine.